### PR TITLE
Feat:-Homepage

### PR DESCRIPTION
**Pull request by shubhamkumar648**
### file to check
1. ecomerce-app/src/component/Navbar/Navbar.jsx
2. ecomerce-app/src/component/banner/Banner.jsx
3. ecomerce-app/src/component/index.js
4. ecomerce-app/src/pages/home/Home.jsx

### CODE REVIEW

File: ecomerce-app/src/component/banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            </h3>
+            <h5 className="text_center headtext_color lh-l font-xxl">
+              We provide Quality Agriculture Products at a reliable price
+            </h5>
+
+            <div className="shopNow">
+              <button className="btn btn__primary m-auto " styles="m-auto">
+                {/* <a href="./pages/productlist.html" className="link__nostyle">
+                  Shop Now
+                </a> */}
```
Comment (rahulcs754) [inline ecomerce-app/src/component/banner/Banner.jsx]: always remove comments

File: ecomerce-app/src/component/categories/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import pesticides from "../../assest/pesticide.jpg"
+import fertilizer from "../../assest/fertilizer.jpg";
+import seed from "../../assest/seed.jpg";
+import tool from "../../assest/tools.jpg";
```
Comment (rahulcs754) [inline ecomerce-app/src/component/categories/Categories.jsx:5]: create one index file and access all file using one like router

File: ecomerce-app/src/pages/home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./Home.css";
+
+
+import { Banner, Categories, Footer, Navbar } from "../../component/index";
```
Comment (rahulcs754) [inline ecomerce-app/src/pages/home/Home.jsx]: you don't need to write index

**Review by rahulcs754 — APPROVED** [review]
just a small suggestion is given in this pr focus on it.

File: ecomerce-app/src/component/banner/Banner.jsx (hunk 1 above)
Comment (shubhamkumar648) [reply ecomerce-app/src/component/banner/Banner.jsx]: will do in next commit

File: ecomerce-app/src/component/categories/Categories.jsx (hunk 2 above)
Comment (shubhamkumar648) [reply ecomerce-app/src/component/categories/Categories.jsx:5]: okay i will do

File: ecomerce-app/src/pages/home/Home.jsx (hunk 3 above)
Comment (shubhamkumar648) [reply ecomerce-app/src/pages/home/Home.jsx]: okay

File: ecomerce-app/src/component/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { FaHeart,FaShoppingCart } from "react-icons/fa";
+
+import  "./Navbar.module.css"
+
+export  function Navbar() {
```
Comment (Gautam2305) [inline ecomerce-app/src/component/Navbar/Navbar.jsx]: use ES6+ here too as you have used it everywhere in your other files. 

My suggestions are also the same as Rahul Shukla.

I really like how you used react-icons ... saw it for the first time.

**Review by Gautam2305 — APPROVED** [review]

File: ecomerce-app/src/component/Navbar/Navbar.jsx (hunk 4 above)
Comment (shubhamkumar648) [reply ecomerce-app/src/component/Navbar/Navbar.jsx]: Thank you! I will change in next commit

**Pull request merged**